README.md

```diff
@@ -1,10 +1,64 @@
-# Tooling
+![Chexxo Logo](assets/logo.png)
+# Chexxo
+Chexxo is a browser extension which enables it's user to identify the quality of the TLS-Certificate which is used to secure the connection between himself and the website he is visiting.
 
-- **Eslint** for linting (finding bugs)
+The extension differentiates three types of qualities as defined by the CA/Browser Forum. Those are in ascending order by quality `domain validated`, `organization validated` and `extended validated`. More information regarding those quality types can be found below.
+
+If the quality of a certificate decreases this is a strong indicated that something fishy is going on which is why we developed this extension in order to allow users to react to such a change.
+
+## Features
+- The certificate quality according to the ones specified by CA/Browser Forum is made visibile by the extension-
+- Details of the certificate can be shown by navigating to the certificate view inside the extension.
+- The extension blocks requests to domains from which the quality has decreased. This blocking behaviour can be activated and deactivated in the extension settings.
+
+## Screenshots
+### Quality view
+![Home](store/assets/firefox/home.png)
+
+### Certificate view
+![Certificate View](store/assets/firefox/certificate.png)
+
+### Certificate qualities
+![Certificate Qualitites](store/assets/firefox/qualities.png)
+
+### Blocked view 
+![Blocked view](store/assets/firefox/blocked.png)
+
+### Settings Firefox 
+![Settings view](store/assets/firefox/settings.png)
+
+### Settings Chromium 
+![Settings view](store/assets/chrome/settings.png)
+
+## Certificate Qualities
+### Domain validated
+If a server maintainer wants to gain a `domain validated` certificate all he has to do is prove that he owns the domain. This may be done by some sort of [challenge](https://letsencrypt.org/docs/challenge-types/). The best known provider of such certificates is `Let's Encrypt`. This type of certificate is also the one most used in phising and malware attacks. Almost all such attacks use this certficate type.
```

Comment:
```suggestion
If a server maintainer wants to gain a `domain validated` certificate all he has to do is prove that he owns the domain. This may be done by some sort of [challenge](https://letsencrypt.org/docs/challenge-types/). The best known provider of such certificates is `Let's Encrypt`. This type of certificate is also the one most used in phishing and malware attacks. Almost all such attacks use this certficate type.
```